service/integration/open_pbs/README.rst

```diff
@@ -50,7 +50,7 @@ The GEOPM power limit hook requires:
 - OpenPBS
 - geopmdpy (and libgeopmd) on nodes where the power limit feature is needed
 - This feature requires platform vendor HW support. In particular, it requires
-  HW support for the MSR::PLATFORM_POWER_LIMIT MSRs (PLATFORM_POWER_LIMIT,
+  HW support for the MSR::PLATFORM_POWER_LIMIT MSRs (PL1_POWER_LIMIT,
   PL1_TIME_WINDOW, PL1_CLAMP_ENABLE, PL1_LIMIT_ENABLE), and the
   MSR::PLATFORM_ENERGY_STATUS MSRs.
```

Comment:
```suggestion
  PL1_TIME_WINDOW, PL1_CLAMP_ENABLE, PL1_LIMIT_ENABLE).
```

I don't see the PLATFORM_ENERGY_STATUS MSR being used anywhere.  I recall we discussed this before, but I don't remember why we left this in.  Was it intentional?